app/models/manageiq/providers/google/inventory/collector/network_manager.rb

```diff
@@ -11,8 +11,8 @@ def cloud_subnets
       @subnetworks += connection.networks.select { |x| x.ipv4_range.present? }.map do |x|
         Fog::Compute::Google::Subnetwork.new(
           :name               => x.name,
-          :gateway_address    => x.gateway_ipv4,
-          :ip_cidr_range      => x.ipv4_range,
+          :gateway_address    => x.gateway_i_pv4,
+          :ip_cidr_range      => x.i_pv4_range,
```

Comment:
While this wasn't necessary, if the first one failed this might in the future so just to be safe

Reply:
Oddly enough this one seems to work.

Reply:
Agreed which does worry me

Reply:
https://github.com/fog/fog-google/issues/478